chromadb/utils/embedding_functions/wordllama_embedding_function.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import annotations
```

Comment:
nit: Is this used?

Reply:
Yes, it allows type-checker to use classes behind `if TYPE_CHECKING` block.

Looks like it becomes default in python 3.14
https://peps.python.org/pep-0649/

And it saves us from 

```
    # Since we do dynamic imports we have to type this as Any
```

like in SentenceTransformers file :)